src/hooks/useImagePalette/useImagePalette.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useContext, useCallback } from 'react';
+import ImageColors, { AndroidImageColors, IOSImageColors } from 'react-native-image-colors';
+import defaultPalette from '../../theme/styles';
+import { ImagePaletteContext, ImagePaletteContextData } from '../../context/ImagePaletteContext';
+
+const useImagePalette = (imageSrc: string) => {
```

Comment:
Create a ticket to refactor this hook into the context, I think we can get away with handling that in there instead of having to call this hook to populate it.

Reply:
See #69